dev-tools/omdb/src/bin/omdb/nexus.rs

```diff
@@ -2595,15 +2604,18 @@ async fn cmd_nexus_blueprints_show(
 
 async fn cmd_nexus_blueprints_diff(
     client: &nexus_client::Client,
-    args: &BlueprintIdsArgs,
+    args: &BlueprintDiffArgs,
 ) -> Result<(), anyhow::Error> {
     let (b1, b2) = try_join(
-        args.blueprint1_id.resolve_to_blueprint(client),
-        args.blueprint2_id.resolve_to_blueprint(client),
+        args.ids.blueprint1_id.resolve_to_blueprint(client),
+        args.ids.blueprint2_id.resolve_to_blueprint(client),
     )
     .await?;
     let diff = b2.diff_since_blueprint(&b1);
     println!("{}", diff.display());
+    if args.exit_code && diff.has_changes() {
+        std::process::exit(1);
+    }
```

Comment:
I like this idea but why not always do this (instead of conditional on an option)?  That's how `diff` works.